src/documents/tasks.py

```diff
@@ -62,6 +70,112 @@ def train_classifier():
         logger.warning("Classifier error: " + str(e))
 
 
+def barcode_reader(image) -> list[str]:
```

Comment:
To support 3.8, I think you'll need to:
```
from typing import List
-> List[int]
```

Don't forget the `pre-commit` hooks, it will sort the imports